ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py

```diff
@@ -49,7 +49,7 @@ def collect(self, device, log):
             'port': device.zCommandPort,
             'user': device.zCommandUsername,
             'password': device.zCommandPassword,
-            'identities': ['~/.ssh/id_rsa', '~/.ssh/id_dsa'],
+            'identities': [device.zKeyPath],
```

Comment:
I don't think this will work unless you also add zKeyPath to nova.deviceProperties first.